src/audio/copier/copier.c

```diff
@@ -1820,6 +1831,7 @@ static const struct comp_driver comp_copier = {
 		.dai_ts_stop			= copier_dai_ts_stop_op,
 		.dai_ts_get			= copier_dai_ts_get_op,
 		.dai_get_hw_params		= copier_get_hw_params,
+		.unbind			= copier_unbind,
```

Comment:
1. I am trying to remove those not compatible interface with module, this is a newly added one.
2. where to call copier_unbind? I did not find in current PR.

Reply:
I think we discussed this and decided to keep them

Reply:
@btian1 1. You probably know better with @ranj063 how the disconnect is handled with module interface (not adapted but native module_interface.h implementation). I think this could be easier as the list of input/outputs is passed to every call to process() so the module anyways has to check for changes on every call. So the converted interface might not need this at all. But as long as we use component, the possibility of disconnect must be handled and here I do it with unbind.

For example module_adapter.c does handle unbind and it modifies the list of buffers passed to module process(). I.e. it covers this scenario.

2. copier_unbind is called from ipc4_unbind_module_instance() -> ipc_comp_disconnect() -> comp_unbind()

Reply:
yes, lets keep it for now. the module adapter might need extensions to handle this